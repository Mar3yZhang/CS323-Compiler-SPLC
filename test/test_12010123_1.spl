int test_1_r06()
{
    int right_id_1, _right_id_2;
    float 3_wrong_id;
    float 3.3_wrong_id;
    float 'c'_wrong_id;
    float _wrong_id3.3;
    float _wrong_id'c';
    float 0x00_woron;
    float _wor3.22on;
}
