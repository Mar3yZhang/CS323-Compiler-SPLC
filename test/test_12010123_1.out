Error type A at Line 4: unknown lexeme 3_wrong_id
Error type A at Line 5: unknown lexeme 3.3_wrong_id
Error type A at Line 6: unknown lexeme 'c'_wrong_id
Error type A at Line 7: unknown lexeme _wrong_id3.3
Error type A at Line 8: unknown lexeme _wrong_id'c'
Error type A at Line 9: unknown lexeme 0x00_woron
Error type A at Line 10: unknown lexeme _wor3.22on